script/LookupEditor.js

```diff
@@ -92,7 +92,8 @@ var LookupEditor = function (idx, table) {
                 $form.serialize()
             )
                 .done(function (data) {
-                    $table.find('tbody').append(data);
+                    const $tbody = $table.find('tbody').length ? $table.find('tbody') : jQuery('<tbody>').appendTo($table);
```

Comment:
This calls $table.find('tbody') twice which should be avoided. I'm also not a fan of the tertiary short hand if-clause.